guides/common/modules/proc_distribution-of-archived-content-view-versions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Distribution_of_Archived_Content_View_Versions_{context}"]
+= Distribution of Archived Content View Versions
+
+If you have a Content View version that is not associated with any lifecycle environments, you can set the default view of that Content View version to be viewed.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings*.
+. Click the *Content* tab.
+. Set the *Distribute archived content view versions* parameter to *Yes*.
+. Click *Submit*.
++
+This enables the repositories of Content View versions without lifecycle environments to be distributed at `/pulp/content/<organization>/content_views/<content view>/X.Y/…`.
++
+If you set the *Distribute archived content view versions* parameter to *No*, it does not delete the old repositories of Content View versions but does not create new repositories.
```

Comment:
The published versions don't change, it only changes what is distributed at `/pulp/content/<organization>/content_views/<content view>/X.Y/…`

Note that older "archived" content view versions won't suddenly become distributed once the setting is enabled. Only new ones will.

Reply:
I can replace the line `If you set the...` with a NOTE that says:
`Older non-promoted Content View versions are not distributed once the setting is enabled.
Only new Content View versions become distributed.`

Reply:
I think that works fine. I think that people debugging should understand the concept of content distribution.